Documentation/compatibility/wpf-KB-focus-now-moves-correctly-across-hosting-scenarios.md

```diff
@@ -0,0 +1,41 @@
+## Keyboard focus now moves correctly across multiple layers of WinForms/WPF hosting 
+
+### Scope
+Edge
+
+### Version Introduced
+4.7.2
+
+### Source Analyzer Status
+NotPlanned
+
+### Change Description
+Consider a WPF application hosting a WinForms control which in turn is hosting WPF controls. Users may not be able to tab out of the WinForms layer if the first or last control in that layer is the WPF ElementHost. This change fixes this issue, and users are now able to tab out of the WinForms layer, but automated applications that rely on focus looping inside the WinForms layer may be affected.
+
+- [x] Quirked
+- [ ] Build-time break
+
+### Recommended Action
+A developer who wants to utilize this change while targeting a framework version below .NET 4.7.2 or a developer who requires the previous functionality while targeting .NET 4.7.2 or greater can set the following set of AppContext flags appropriately.  Values of false for all of them will result in this change being enabled.
```

Comment:
It's best to do this in two parts, with two separate fenced code blocks. The first for apps broken by the change who want to opt out of it. The second for downlevel apps running under 4.7.2 that want to opt in.
It's unclear what "all of them" means. Presumably it's the two switches, but they haven't been introduced yet.